pmg/bills.py

```diff
@@ -111,9 +111,61 @@ def bill_history(bill):
         info.update(location)
         history.append(info)
 
+    history = hansard_linking(history)
     return history
 
 
+def match_title(event_title):
+    """
+    Match bill title against the following possible titles
+    """
+    bill_titles = [
+        "Bill passed by National Assembly",
+        "Bill passed by both Houses",
+        "Bill revived on this date",
+        "The NCOP rescinded",
+        "Bill remitted",
+    ]
+    for title in bill_titles:
```

Comment:
another way of writing this is with `any(lambda ...)`
https://docs.python.org/3/library/functions.html#any

Not a change request, just a hint for when it makes sense. It's clear enough here